backend/settings.py

```diff
@@ -218,6 +219,11 @@
         "default_link_target": "_blank",
     }
 
+# Watchman settings (https://github.com/mwarkentin/django-watchman)
+WATCHMAN_TOKENS = os.getenv('DJANGO_WATCHMAN_TOKENS', None)
+WATCHMAN_TOKEN_NAME = os.getenv('DJANGO_WATCHMAN_TOKEN_NAME', 'token')
+WATCHMAN_CHECKS = ('watchman.checks.caches', 'watchman.checks.databases')
```

Comment:
This is defined in order to exclude the default `watchman.checks.storage`, which is failing due to our use of [`django-db-file-storage`](https://pypi.org/project/django-db-file-storage/). There may be a way to fix this, but it would require a custom `django-watchman` check. I'm also not sure how useful the storage check is if we are ultimately storing the file content in the database; maybe the database check is enough.

Reply:
I think this is ok. We can add this `storage` later iteration if needed.